src/code/z_jpeg.c

```diff
@@ -252,6 +252,7 @@ s32 Jpeg_Decode(void* data, void* zbuffer, void* work, u32 workSize) {
             break;
 
         case 4:
+            // This chained if-else have printfs in them on debug
```

Comment:
```suggestion
            // This chained if-else has printfs inside it on debug
```
Or "These chained if-elses have printfs inside them on debug"